deploy/examples/create-external-cluster-resources.py

```diff
@@ -1134,7 +1142,24 @@ def create_rgw_admin_ops_user(self):
                     % (cmd, execErr.output, execErr.returncode, execErr.stderr)
                 )
                 raise Exception(err_msg)
-
+            if self._arg_parser.rgw_realm_name:
+                cmd = [
+                    "radosgw-admin",
+                    "user",
+                    "modify",
+                    "--uid",
+                    self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME,
+                    "--realm-new-name",
+                    self._arg_parser.rgw_realm_name,
+                ]
+                try:
+                    output = subprocess.check_output(cmd, stderr=subprocess.PIPE)
+                except subprocess.CalledProcessError as execErr:
+                    err_msg = (
+                        "failed to execute command %s. Output: %s. Code: %s. Error: %s"
+                        % (cmd, execErr.output, execErr.returncode, execErr.stderr)
+                    )
```

Comment:
nit: consider using f-strings for formatting:

```suggestion
                except subprocess.CalledProcessError as e:
                    err_msg = (
                    f"failed to execute command {cmd}. Output: {e.output}. Code: {e.returncode}. Error: {estderr}"
                    )
```

Reply:
Thanks maybe I can open a separate issue for this,
And would apply to overall code.